tests/hardhat/Fork/RiskFund.ts

```diff
@@ -44,13 +44,16 @@ let whitePaperRateFactory: WhitePaperInterestRateModelFactory;
 let fakeAccessControlManager: FakeContract<AccessControlManager>;
 let protocolShareReserve: ProtocolShareReserve;
 let riskFund: RiskFund;
-let pancakeSwapRouter: PancakeRouter;
+let pancakeSwapRouter: PancakeRouter | MockContract<PancakeRouter>;
```

Comment:
I think the Mock contract is preferable in general because it allows for real contract code to be used by default rather than mocking the entire contract